fmlearlydisplay/src/main/java/net/minecraftforge/fml/earlydisplay/ColourScheme.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.fml.earlydisplay;
+
+public enum ColourScheme {
```

Comment:
Mojang uses a mix, because Colour and Color are synonyms, based on the language of the author. I am British and spell it Colour. This will remain Colour.